test/utils/image_utils.py

```diff
@@ -16,35 +16,48 @@
 import subprocess
 import sys
 
-CYAN_COLOR = '\033[36m'
-END_COLOR = '\033[0m'
-DLC_AWS_ID = '763104351884'
+CYAN_COLOR = "\033[36m"
+END_COLOR = "\033[0m"
+DLC_AWS_ID = "763104351884"
 
 
-def build_image(framework_version, dockerfile, image_uri, region, cwd='.'):
-    _check_call('python setup.py sdist')
+def build_image(framework_version, dockerfile, image_uri, region, cwd="."):
+    _check_call("python setup.py sdist")
 
-    if 'dlc' in dockerfile:
+    if "dlc" in dockerfile:
         ecr_login(region, DLC_AWS_ID)
 
-    dockerfile_location = os.path.join('test', 'container', framework_version, dockerfile)
+    dockerfile_location = os.path.join("test", "container", framework_version, dockerfile)
 
     subprocess.check_call(
-        ['docker', 'build', '-t', image_uri, '-f', dockerfile_location, '--build-arg',
-         'region={}'.format(region), cwd], cwd=cwd)
-    print('created image {}'.format(image_uri))
+        [
+            "docker",
+            "build",
+            "-t",
+            image_uri,
+            "-f",
+            dockerfile_location,
+            "--build-arg",
+            "region={}".format(region),
+            cwd,
+        ],
+        cwd=cwd,
+    )
+    print("created image {}".format(image_uri))
     return image_uri
 
 
 def push_image(ecr_image, region, aws_id):
     ecr_login(region, aws_id)
-    _check_call('docker push {}'.format(ecr_image))
+    _check_call("docker push {}".format(ecr_image))
 
 
 def ecr_login(region, aws_id):
-    login = _check_call('aws ecr get-login --registry-ids {} '.format(aws_id)
-                        + '--no-include-email --region {}'.format(region))
-    _check_call(login.decode('utf-8').rstrip('\n'))
+    login = _check_call(
+        "aws ecr get-login --registry-ids {} ".format(aws_id)
+        + "--no-include-email --region {}".format(region)
+    )
```

Comment:
You may want to move to using the new `aws ecr get-login-password` command, rather than the old `aws ecr get-login` command, which will be deprecated soon.

https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login.html#description

https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ecr/get-login-password.html

Reply:
Will fix this in another PR as well.